PendingReleaseNotes.md

```diff
@@ -8,6 +8,7 @@
 - Rook-Operator no longer creates the resources CRD's or TPR's at the runtime. Instead, those resources are provisioned during deployment via `helm` or `kubectl`.
 
 ## Breaking Changes
+- Removed support for Kubernetes 1.6, including the legacy Third Party Resources (TPRs).
```

Comment:
I think changing `1.6` to `<= 1.6` (or something like that) is better as 1.5 support is also finally dropped.

Reply:
sure, i'll make that more clear thanks